functions/exercise21/scriptSpec.js

```diff
@@ -0,0 +1,17 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals getPossibleSubset*/
+
+describe("Getting all possible subset with a fixed length combinations in an array", (() => {
+    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2]",  (() => {
+        var subsets = getPossibleSubset([1, 2, 3], 2);
+        expect(subsets.elementOfsubset).toEqual([
+            [3, 2],
+        ]);
+        expect(subsets.elementOfsubset).toEqual([
```

Comment:
What is elementOfsubset? In previous expectetion you expect it to be equal [3,2] and here you expect it to be equal [3,1], so you epxect:
elementOfSubset = [3,1] = [3,2]
So you expect that [3,1] = [3,2], which is not equal, so this assumption is wrong.